### PR TITLE
react-bootstrap-datetimepicker

### PR DESCRIPTION
**Pull request by gregegan**
https://www.npmjs.com/package/react-bootstrap-datetimepicker

### CODE REVIEW

**Comment by marudor** [comment]
Just like your other PRs - could you fix the names?

**Comment by thejameskyle** [comment]
Yup the two files you should have are these:

```
/react-bootstrap-datetimepicker_v0.0.x/flow_>=v0.30.x/react-bootstrap-datetimepicker_v0.0.x.js.flow
/react-bootstrap-datetimepicker_v0.0.x/test_react-bootstrap-datetimepicker_v0.0.x.js.flow
```

The second is for adding tests to have a sanity check.

**Comment by chollier** [comment]
Just a side note as I was stumbling around that repo looking for type definitions, I'm the orignal author of `react-bootstrap-datetimepicker` but I recently deprecated it. It will likely not get any updates or support for newer react version and I'm redirecting users to use https://github.com/YouCanBookMe/react-datetime

TL;DR : you can probably close this issue
